realm/realm-library/src/objectServer/java/io/realm/SyncUser.java

```diff
@@ -462,6 +477,15 @@ private static String getManagementRealmUrl(URL authUrl) {
         }
     }
 
+    /**
+     * FIXME Javadoc
+     *
+     * @return
+     */
+    public PermissionManager getPermissionManager() {
```

Comment:
Because the PermissionManager is strongly tied to two underlying Realms. As long as they are open, the user cannot call logout and hopefully people would be used to this pattern if using Realm already. I agree it isn't as nice as one could hope. 

The other alternative would be to make `close()` private and then have a finalizer call it. We would also need to determine how `syncuser.logout()` would behave in that case. I would assume it would just forcibly close the PermissionManager. Right now it would throw an error if you tried to logout while the permissionmanager was open.

What do you think?